src/components/Calculator.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import './Calculator.css';
+import './calculator.css';
 
 function Pad() {
```

Comment:
* Kindly use class-based components for all your components as [required  ](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#:~:text=Use%20class%20based%20components%20(All%20components%20should%20be%20class%2Dbased).)